MAUI/DataForm/Overview.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Overview of .NET MAUI DataForm control | Syncfusion
+description: Learn about the introduction of Syncfusion .NET MAUI DataForm(SfDataForm) control and its key features.
```

Comment:
Learn about the introduction of Syncfusion .NET MAUI DataForm(SfDataForm) control and its key features in mobile and desktop apps from a single shared codebase.

Reply:
changed